Header header
float64 time_increment
uint32 object_id
uint32 object_label
geometry_msgs/Point point

geometry_msgs/Point acc
geometry_msgs/Point vel

